### DISCUSSION TITLE
Make the profile overlay select the same mode you've selected in-game

### DISCUSSION THREAD

**Discussion by Emanbg** [ppy/osu#30404] [Ideas]
When hovering on the account panel your rank updates according to the selected mode but when opening the profile page it defaults to the Main Mode the person has selected for their account. This would be a nice QoL feature for people who play multiple modes.

I apologize for the "low effort" this is just a very minor change and my first Issue/Discussion. 

Using latest build: 2024.1009.1
https://github.com/user-attachments/assets/ae9c2c97-84ae-42c0-ae80-da68b6333d74

**Comment by pigeonskew** [comment]
Wishing this gets implemented.

**Comment by n0Oo0Oo0b** [comment]
Duplicate of #24399, but I think we can keep this thread instead since this is worded a lot better.
